src/commands/init.js

```diff
@@ -91,7 +92,19 @@ class InitCommand extends AbstractAuthenticatedCommand {
         const databaseConfiguration = await handleDatabaseConfiguration();
         this.spinner.continue();
         if (databaseConfiguration) {
-          this.environmentVariables.databaseUrl = buildDatabaseUrl(databaseConfiguration);
+          const dbConfig = {
```

Comment:
Why rename here, and not in the database-prompts.js file?

Is something preventing us from using the same name everywhere?

Reply:
This is an even bigger refactor, which ultimately should be done, but we decided it was outside the scope here.

Reply:
Totally agree with you @romain-gilliotte, but with @DrRaider as-well.